internal/lightcone/abundance/timewaitsfornoone/timewaitsfornoone.go

```diff
@@ -0,0 +1,95 @@
+package timewaitsfornoone
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	time = "time-waits-for-no-one"
+	cd   = "time-waits-for-no-one-extra-dmg-cd"
```

Comment:
nit: would just name this `time-waits-for-no-one-cd` since the dmg is the only thing that can go on cd